tests/Unit/Classes/DscResources/nxService.tests.ps1

```diff
@@ -5,9 +5,7 @@ $script:testService = "myTestService.service" # Mock service for testing
 Describe "nxService resource for managing services on a Linux node" {
     BeforeAll {
         Mock -ModuleName 'nxtools' -CommandName 'Invoke-NativeCommand' -ParameterFilter {
-            $expected = @('is-enabled', $testService)
-            $diff = Compare-Object $Parameters $expected
-            return $Executable -eq "systemctl" -and $diff.Count -eq 0
+            return $Executable -eq "systemctl" -and $Parameters -contains "is-enabled"
```

Comment:
Taking out the exact compare makes this check different? Before $Parameters had to be exactly @($testService, is-enabled)
Now it can be anything as long as it has is-enabled?
Just making sure this is intentional and not a test regression?

Reply:
This was intentional. I needed to update this mock to cover the new unit test where the service is an empty string. I fixed this in the latest commit so instead of having a vague mock, I have two explicit mocks. I think it's clearer.